Code/RDBoost/Wrap/RDBase.cpp

```diff
@@ -235,6 +235,80 @@ struct python_ostream_wrapper {
 };
 
 void seedRNG(unsigned int seed) { std::srand(seed); }
+
+/// Simple Boost.Python custom converter from pathlib.Path to std::string
+template <typename T = std::string>
+struct path_converter {
+  path_converter() {
+    python::converter::registry::push_back(&path_converter::convertible,
+                                           &path_converter::construct,
+                                           boost::python::type_id<T>());
+  }
+
+  /// Check PyObject is a pathlib.Path
+  static void *convertible(PyObject *object) {
+    // paranoia
+    if (object == nullptr) {
+      return nullptr;
+    }
+    python::object boost_object(python::handle<>(python::borrowed(object)));
+
+    std::string object_classname = boost::python::extract<std::string>(
+        boost_object.attr("__class__").attr("__name__"));
+    // pathlib.Path is always specialized to the below derived classes
+    if (object_classname == "WindowsPath" || object_classname == "PosixPath") {
+      return object;
+    }
+
+    return nullptr;
+  }
+
+  /// Construct a std::string from pathlib.Path using its own __str__ attribute
+  static void construct(
+      PyObject *object,
+      boost::python::converter::rvalue_from_python_stage1_data *data) {
+    void *storage =
+        ((boost::python::converter::rvalue_from_python_storage<T> *)data)
+            ->storage.bytes;
+    python::object boost_object{python::handle<>{python::borrowed(object)}};
+    new (storage)
+        T{boost::python::extract<std::string>{boost_object.attr("__str__")()}};
+    data->convertible = storage;
+  }
+};
+
+/// Convert a Python str to a std::string_view
+template <typename T = std::string_view>
+struct string_view_converter {
+  string_view_converter() {
+    python::converter::registry::push_back(&string_view_converter::convertible,
+                                           &string_view_converter::construct,
+                                           boost::python::type_id<T>());
+  }
+
+  /// Check PyObject is a str
+  static void *convertible(PyObject *object) {
+    if (object != nullptr && PyUnicode_Check(object)) {
+      return object;
+    }
+    return nullptr;
+  }
+
+  /// Construct a std::string_view from the internal string of the PyObject.
+  /// This shouldn´t fail, as we
```

Comment:
Missing some text here

Reply:
Yup, I was interrupted while writing it, in mid sentence.

<img width="384" height="192" alt="image" src="https://github.com/user-attachments/assets/e7c05f9c-b28e-4605-a56b-c0209816ec99" />